code/datums/loadout/loadout_helper.dm

```diff
@@ -174,3 +174,12 @@
 		return
 	var/datum/loadout/loadout = jatum_deserialize(loadout_json)
 	return loadout
+
+/proc/convert_loadouts_list(list/loadouts_data)
+	var/list/new_loadouts_data = list()
+	for(var/i = 1; i <= length(loadouts_data); i += 2)
+		var/next_loadout_data = list()
+		next_loadout_data += loadouts_data[i]
+		next_loadout_data += loadouts_data[i+1]
```

Comment:
use ++

Reply:
that means step 1 or i+=1 in the loop description, which overall will be less clear IMO

Reply:
No i mean the i+1 in brackets